test/views/coronavirus_local_restrictions_test.rb

```diff
@@ -0,0 +1,90 @@
+require "test_helper"
+require "gds_api/test_helpers/mapit"
+
+class CoronavirusLocalRestrictionTest < ActionView::TestCase
+  include GdsApi::TestHelpers::Mapit
+
+  test "renders postcode match tier 4" do
```

Comment:
This might need a little more information. It seems a little unclear to me quite what this is testing. is it `test "rendering tier 4 guidance for a postcode in tier 4"` ?

Reply:
thanks, I updated